### PR TITLE
Feature/rChain persist results

### LINKED ISSUE #638: rChain : persist results
Write results by uploading CSV file

### PR DESCRIPTION
**Pull request by minotogna**
Resolves #638

### CODE REVIEW

File: server/modules/analysis/manager/processingChainManager.js [hunk 1]
```diff
@@ -46,26 +45,27 @@ const _insertChain = async (user, surveyId, chain, t) => {
...
         ActivityLog.type.processingChainPropUpdate,
         { [ActivityLog.keysContent.uuid]: processingChainUuid, key, value },
         false,
-        t,
+        t
       ),
     ])
-  }
+    .flat()
+  await Promise.all(promises)
```
Comment (SteRiccio) [inline server/modules/analysis/manager/processingChainManager.js:61]: You can even inline promises...

File: server/modules/analysis/service/rChainService/rFile/rFile.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as FileUtils from '@server/utils/file/fileUtils'
 
 import { arenaInfo } from '../rFunctions'
 
-const _lineSeparator = `${StringUtils.NEW_LINE}${StringUtils.NEW_LINE}`
+const _doubleFeed = `${StringUtils.NEW_LINE}${StringUtils.NEW_LINE}`
```
Comment (SteRiccio) [inline server/modules/analysis/service/rChainService/rFile/rFile.js:6]: What about contentSeparator? It tells what it does...

Comment (minotogna) [reply server/modules/analysis/service/rChainService/rFile/rFile.js:6]: double feed no?

File: server/modules/analysis/manager/processingChainManager.js (hunk 1 above)
Comment (minotogna) [reply server/modules/analysis/manager/processingChainManager.js]: ok done

File: server/modules/analysis/service/rChainService/rFile/rFile.js (hunk 2 above)
Comment (minotogna) [reply server/modules/analysis/service/rChainService/rFile/rFile.js]: ok

Comment (minotogna) [reply server/modules/analysis/service/rChainService/rFile/rFile.js]: ok done

Comment (minotogna) [reply server/modules/analysis/service/rChainService/rFile/rFile.js]: ok done

**Pull request merged**